src/main/java/net/daporkchop/toobeetooteebot/client/handler/incoming/entity/EntityMetadataHandler.java

```diff
@@ -46,9 +48,17 @@ public boolean apply(@NonNull ServerEntityMetadataPacket packet, @NonNull PorkCl
                         continue MAINLOOP;
                     }
                 }
-                entity.getMetadata().add(metadata);
+
+                if(entity.getMetadata() instanceof ArrayList) {
+                    entity.getMetadata().add(metadata);
+                } else if(CONFIG.log.sendWarning) {
+                    // TODO make that info less
+                    CLIENT_LOG.warn("Received ServerEntityMetadataPacket for an entity that cant accept those (probably happened because you are on a 1.16 server. You can't see the sneaking of other players now eg)");
```

Comment:
this is a really hacky workaround, for a variety of reasons:

- pork2b2tbot doesn't support 1.16. if you have a custom fork that adds support then this belongs on that fork
- this should be fixed by changing the list implementation used to be an `ArrayList`. there's no reason to work around such a trivial fix